build.gradle.kts

```diff
@@ -312,6 +312,7 @@ tasks {
     register<NpmTask>("npmRunBuild") {
         dependsOn("npmInstall")
         npmCommand.set(listOf("run", "build"))
+        npmCommand.set(listOf("run", "test"))
```

Comment:
I think we need a separate Gradle task for the frontend tests. Something like:

```    register<NpmTask>("npmRunTest") {
        dependsOn("npmRunBuild")
        npmCommand.set(listOf("run", "test"))

        // avoid running this task when there are no changes in the input or output files
        // see: https://github.com/node-gradle/gradle-node-plugin/blob/master/docs/faq.md
        inputs.files(fileTree("src/main/app/node_modules"))
        inputs.files(fileTree("src/main/app/src"))
        inputs.file("src/main/app/package.json")
        inputs.file("src/main/app/package-lock.json")
        outputs.dir("src/main/app/dist/zaakafhandelcomponent")
    }
 ```
    
because otherwise we can never run the tests separately from the build which we may want to do in future (e.g. the speed up our gitHub pipeline and such).

And then we probably want to run this task as part of the overall Gradle test task:

```
  test {
        dependsOn("npmRunTest")
    }
```